examples/Workflow/WorkflowWebApp/Activities/ReserveInventoryActivity.cs

```diff
@@ -1,32 +1,60 @@
 ﻿namespace WorkflowWebApp.Activities
 {
     using System.Threading.Tasks;
+    using Dapr.Client;
     using Dapr.Workflow;
-
-    record InventoryRequest(string RequestId, string Name, int Quantity);
-    record InventoryResult(bool Success);
+    using WorkflowWebApp.Models;
 
     class ReserveInventoryActivity : WorkflowActivity<InventoryRequest, InventoryResult>
     {
         readonly ILogger logger;
+        readonly DaprClient client;
+        private static readonly string storeName = "statestore";
```

Comment:
Looks like this is defined in two places. Let's put it somewhere more global to reduce the duplication.